.travis.yml

```diff
@@ -1,14 +1,14 @@
-language: node_js
-node_js:
-- "4"
-- "node"
 sudo: false
+language: node_js
 cache:
   directories:
-  - node_modules
-install:
-- npm install
-- npm update
+    - node_modules
+node_js:
+  - "4"
+  - "node"
+before_install:
+  - npm update
+  - npm prune
```

Comment:
Moving `npm update` into `before_install` might help detect problems where a package update breaks the `npm run build` automatically triggered by `npm install`.